creativecloud/features/interactive-elements/interactive-elements.js

```diff
@@ -0,0 +1,97 @@
+import { setLibs } from '../../scripts/utils.js';
+
+const miloLibs = setLibs('/libs');
+const { createTag } = await import(`${miloLibs}/utils/utils.js`);
+const interactiveCss = await import('./interactive-elements.css', {
+  assert: { type: 'css' }
+  });
+document.adoptedStyleSheets = [interactiveCss.default];
+
+const DESKTOP_SIZE = 1200;
+const MOBILE_SIZE = 600;
+
+function defineDeviceByScreenSize() {
+  const screenWidth = window.innerWidth;
+  if (screenWidth >= DESKTOP_SIZE) {
+    return 'DESKTOP';
+  }
+  if (screenWidth <= MOBILE_SIZE) {
+    return 'MOBILE';
+  }
+  return 'TABLET';
+}
+
+export function createPromptField(prompt, buttonText, mode, trackingValue = '') {
+  const promptField = createTag('div', { id: 'promptbar', class: 'promptbar' });
+  let promptInput = '';
+  if (mode !== 'genfill') {
+    promptInput = createTag('input', { class: 'promptText', id: 'promptinput', placeholder: `${prompt}`, autofocus: 'true', maxlength: '250' });
+  }
+  const promptButton = createTag('button', { class: 'con-button blue button-xl button-justified-mobile', id: 'promptbutton', 'daa-ll': trackingValue }, `${buttonText}`);
+
+  if (mode === 'light') {
+    promptField.classList.add('light');
+    promptInput.classList.add('light');
+  } else if (mode === 'genfill') {
+    promptButton.setAttribute('id', 'genfill');
+    promptField.classList.remove('promptbar');
+  }
+  if (mode !== 'genfill') {
+    promptField.append(promptInput);
+  }
+  promptField.append(promptButton);
+
+  const device = defineDeviceByScreenSize();
+  if (device === 'TABLET') {
+    promptButton.classList.remove('button-xl');
+    if (!promptButton.classList.contains('button-l')) {
+      promptButton.classList.add('button-l');
+    }
+  } else if (device === 'MOBILE') {
+    promptButton.classList.remove('button-xl');
+    promptButton.classList.remove('button-l');
+  } else if (device === 'DESKTOP') {
+    if (!promptButton.classList.contains('button-xl')) {
+      promptButton.classList.add('button-xl');
+    }
+    promptButton.classList.remove('button-l');
+  }
+  return promptField;
+}
+
+export function createEnticement(enticementDetail, mode) {
+  const enticementDiv = createTag('div');
+  const svgImage = createTag('img', { class: 'enticementArrow', alt: '' });
+  let arrowText;
+  [arrowText, svgImage.src] = enticementDetail.split('|');
```

Comment:
Just a question : Did we consider creating Try It a link that points to the SVG so that we do not have to split and process?

Reply:
Actually we got the approval on authoring experience where the text and svg is separated by pipe. Hence this approach in code.